.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
     - export PATH=${HOME}/edm/bin:${PATH}
     - edm install --version 3.5 -y click setuptools
```

Comment:
Just noticed we should be using `--version 3.6` here

Reply:
Thank you, I will check all the other plugins for that as well.

Reply:
There are other CI updates that were needed in https://github.com/force-h2020/force-bdss-plugin-gromacs/pull/23 that we should also implement to all FORCE repos